### DISCUSSION TITLE
Coupling Neuromancer and Modelica

### DISCUSSION THREAD

**Discussion by weberscode** [pnnl/neuromancer#176] [Ideas]
Hey all, 

is it intended to enable the coupling of Neuromancers differential machine learning models to Modelica system simulations? E.g. like SMArtIInt  https://github.com/xrg-simulation/SMArtIInt does with TF lite models?

I would like to use that with Modelons MA solver.

Kr
Simon

**Comment by drgona** [comment]
Dear @weberscode, we don't plan to integrate Neuromancer with Modelica at the moment.
I would be happy to consult anyone who would like to do so.
